src/components/Questions/Body.vue

```diff
@@ -365,6 +371,16 @@ export default defineComponent({
 
     if (isQuestionImagePresent.value) startImageLoading();
 
+    onMounted(() => {
+      // Force render any math on the page when component is mounted
+      if ('MathJax' in window) (window.MathJax as any).typeset();
+    });
+
+    onUpdated(() => {
```

Comment:
for the player, is there any dynamic update that is happening beyond the mount?

Reply:
when you click continue and the question/option updates, `onUpdated` is called